addons/binding/org.openhab.binding.exec/README.md

```diff
@@ -16,31 +16,42 @@ Note that the commands are executed in the context and with the privileges of th
 
 The `command` Thing requires the command to execute on the shell. Optionally one can specify:
 
-- a transformation to apply on the execution result, 
-- an interval, in seconds, the command will be repeatedly executed, 
-- a time-out, in seconds, the execution of the command will time out, and lastly, 
-- a boolean parameter to make the command execute immediately every time the state of the input channel has changed. 
+- [transform] a transformation to apply on the execution result, 
+- [interval] an interval, in seconds, the command will be repeatedly executed, 
+- [timeput] a time-out, in seconds, the execution of the command will time out, and lastly, 
+- [runOnInput] a boolean parameter to make the command execute immediately every time the state of an input channel is set by a Command 
+- [repeatEnabled] a boolean parameter to allow the command execute repeatedly, e.g. when the state of an input channel is set repeatedly with the same state. This parameter only makes sense when used in combination with [runOnInput] set to true.
```

Comment:
I agree, "state" is not an optimal wording here. Use cases : I can think for example of Dimmers that send a series of INCREASE/DECREASE for example.